test/fixtures/output/asyncapi/asyncapi-spec.yaml

```diff
@@ -21,8 +21,7 @@ channels:
       kafka:
         x-configs:
           cleanup.policy: delete
-          delete.retention.ms: 8.64e+07
-          confluent.value.schema.validation: "true"
+          delete.retention.ms: "86400000"
```

Comment:
Should this be an integer? Why does it use quotes?

Reply:
Every value is stored as a string only. While importing the YAML file, we need to pass it as string only while updating/creating a topic. So, there's no point of converting it on both sides.